### ISSUE TITLE
dvc remove CLI documentation inconsistency

### ISSUE THREAD

**Issue by casperdcl** [iterative/dvc#2456]
`dvc remove` (without `targets`) prints help which states that `targets` are optional, and if not specified will remove all DVC-files. Clearly not the case.

```bash
$ dvc remove
[...]
  targets        DVC-files to remove. Optional. (Finds all DVC-files in the
                 workspace by default.)
```

**Comment by efiop** [comment]
@jorgeorpinel This slipped-in in 936f941bf and will require doc modification. Could you take a look, please?

**Comment by jorgeorpinel** [comment]
Thanks @casperdcl and yes @efiop, fixed in 80076d88.
